### PR TITLE
Patient Record layout update 1

### PR DESCRIPTION
**Pull request by mdsupport**
Updates patient data display as a follow up to #4928.

1. Patient menu can provide patient record data for use by javascript.
2. The content from nav-link targets is now loaded in a common div eliminating need for each script to render menu.

### CODE REVIEW

**Comment by mdsupport** [comment]
This change enables transfer of links from current scripts to single sub-menu.

An example included moves the patient delete button from form to Action -> Delete. Use of forms in dropdowns can also simplify workflow where staff currently is expected to 
1. fetch patient record, 
2. wait for all ajax data to load, 
3. Click edit button, 
4. Clicks the specific tab
5. Updates a field
6. Save the form.

Sample images
image


image

**Comment by sjpadgett** [comment]
@robertdown How does this fit with the refactors you're doing?
I kind of like the idea of making some of the workflows, actions. Anything to speed load times etc...

**Comment by robertdown** [comment]
* During load, the menu gets displayed, then, after the js runs it gets repainted in the final rendering. This is jarring and moves elements on the page which will lead to errors.
* The styling of the page sacrifices the UX of the computer user for the UX of the mobile user. While I appreciate the need for mobile compatibility, we should not trade off the computer user for the mobile user. White space and padding has value (even on mobile)
* The UI of the menu does not flow with the rest of the styling
* Inactive tooltip isn't displaying the right data in my test

`displayHorizNavBarMenu()` hasn't been moved to Twig yet, but if we are going to refactor it, we should go ahead and use a proper Twig template. Other comments in a proper review

File: interface/patient_file/summary/demographics.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/* Reformat EMR Patient Record Display
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    MD Support<mdsupport@users.sourceforge.net>
+ * @copyright Copyright (c) 2021-2022 MD Support<mdsupport@users.sourceforge.net>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+*/
+
+$('body')
```
Comment (robertdown) [inline interface/patient_file/summary/demographics.js:9]: Why is all of this occurring in jQuery and not just added to the HTML class?

**Review by robertdown — CHANGES_REQUESTED** [review]

File: interface/patient_file/summary/demographics.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+.addClass('text-body py-1')
+.removeClass('text-white');
+
+// Replace generic Dashboard label by patient name
+try
+{
+    let objPt = $('#dashboard').data('ptdata');
+    let disp = objPt.lfname;
+    if (objPt.deceased_date !== null) {
+        disp += `<i class="fas fa-exclamation-triangle ml-1 bg-light text-warning" data-toggle="tooltip" title="${objPt.inactive_days}"></i>`;
```
Comment (robertdown) [inline interface/patient_file/summary/demographics.js:34]: bg-light is breaking the UI here

**Pull request closed without merging**